test/test_api/test_interpreter.py

```diff
@@ -532,7 +532,6 @@ def func(a, b, c):
 
 def test_type_var():
     """This was an issue before, see Github #1369"""
-    import typing
```

Comment:
I'm _assuming_ that the inline nature of this import is immaterial; I can't see anything in the origin of this test (https://github.com/davidhalter/jedi/commit/ab80646b860d89f0fea2c3aea262cd922c7413d7) which suggests that a global import (as this file now has) would change the behaviour, but would be good to get a second opinion.